Journalator_Display/Source/Display/InvestONator/PortfolioDisplay.lua

```
@@ -0,0 +1,426 @@
+  self:SetupCreatePortfolioDialog()
+  self:RefreshPortfolioList()
+end
+
+---Handle the display being shown
+---Refreshes the portfolio list
+function JournalatorInvestONatorPortfolioDisplayMixin:OnShow()
+  self:RefreshPortfolioList()
+end
+
+---Set up the scrollable portfolio list
+function JournalatorInvestONatorPortfolioDisplayMixin:SetupPortfolioList()
```

Luacheck (1.2.0)

[warning] mutating non-standard global variable 'JournalatorInvestONatorPortfolioDisplayMixin'

(W112)